tests/eclient/testdata/dev_local_info.txt

```diff
@@ -0,0 +1,241 @@
+# Test dev local info
+
+{{define "mngr_port"}}8027{{end}}
+{{define "app_port"}}8028{{end}}
+{{define "token"}}server_token_123{{end}}
+{{define "app_info_status_file"}}/mnt/app-info-status.json{{end}}
+{{define "dev_info_status_file"}}/mnt/dev-info-status.json{{end}}
+{{define "dev_cmd_file"}}/mnt/dev-command.json{{end}}
+{{define "network"}}n1{{end}}
+{{define "ssh"}}ssh -q -o ConnectTimeout=10 -o StrictHostKeyChecking=no -o PasswordAuthentication=no -i {{EdenConfig "eden.tests"}}/eclient/image/cert/id_rsa root@{{end}}
+
+[!exec:bash] stop
+[!exec:sleep] stop
+[!exec:ssh] stop
+[!exec:chmod] stop
+[!exec:jq] stop
+
+exec chmod 600 {{EdenConfig "eden.tests"}}/eclient/image/cert/id_rsa
+
+# Starting of reboot detector with a 2 reboot limit
+! test eden.reboot.test -test.v -timewait 100m -reboot=0 -count=2 &
```

Comment:
It should catch 3 reboots to fail (2 reboots are expected, the third one is an error).

Reply:
OK, didn't realize the limit had that semantics; thought it was the expected number not the number at which it is a failure.